adapters/jsdom-svelte/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import {
+    getDefinitionForKeyType,
+    isMultipleElementsWithLocatorError,
+    Locator,
+    MultipleElementsWithLocatorError,
+    NoElementWithLocatorError,
+    UniDriver, UniDriverList, waitFor
+} from "@unidriver/core";
+
+import {fireEvent} from '@testing-library/svelte';
+
+type HTMLFocusableElement = HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement | HTMLButtonElement;
```

Comment:
`HTMLAnchorElement` is also focusable

Reply:
worth to add to react also maybe?

Reply:
Lets ask @GabiGrin, changing react adapter might break existing tests

Reply:
@nktssh  @giladsegal but then it means it _should_ break, no? 🤔